src/utils/utils.ts

```diff
@@ -299,3 +301,23 @@ export const oppfolingsdatoEnsligeForsorgere = (alderBarn?: Date) => {
     const formatertDato = toDatePrettyPrint(datoBarnEttAar);
     return `${formatertDato} (Barn 1 år)`;
 };
+
+export const oppdaterBrukerIKontekstOgNavigerTilLenke = (
+    fnr: string,
+    lenke: string,
+    onSuksess: () => void,
+    onFeilet: () => void
+) => {
+    settBrukerIKontekst(fnr)
+        .then(() => {
+            onSuksess();
+            window.location.href = lenke;
+        })
+        .catch(onFeilet);
+};
+
+export const vedKlikkUtenfor = (refs: RefObject<HTMLElement>[], klikkTarget: Node | null, fn: () => void) => {
+    if (!refs.some(ref => ref.current?.contains(klikkTarget))) {
```

Comment:
this is hard to understand, especially with variable names like this.

Reply:
should it be some default behaviour that popup closes when someone clicks outside..

Reply:
I agree that the variable naming could be better.

The behaviour is not that clicks outside the popup should close the popup, bur rather clicking outside of either the popup or the button that opens it should remove the button _and_ the popup (it's a bit hard to describe this behaviour).